### PR TITLE
Make customer repr account aware

### PR DESCRIPTION
**Pull request by ticosax**

### CODE REVIEW

File: pinax/stripe/models.py [hunk 1]
```diff
@@ -285,14 +285,26 @@ def stripe_customer(self):
         )
 
     def __str__(self):
-        return str(self.user)
+        if self.user:
+            return str(self.user)
+        elif self.id:
+            return ", ".join(repr(user) for user in self.users.all())
+        return "None"
```
Comment (blueyed) [inline pinax/stripe/models.py]: s/None/No user(s)/ ?

**Comment by blueyed** [comment]
Would be nice to have this for helping with debugging.

**Comment by ticosax** [comment]
Will do, I forgot

**Review by blueyed — APPROVED** [review]
Thanks!

**Pull request merged**